layouts/partials/navbar.html

```
@@ -0,0 +1,63 @@
+{{- $title          := .Site.Params.title }}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)